index.html

```diff
@@ -114,6 +114,41 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+          <!-- ________ *Nicholas' card START ________  -->
+    <div class="card">
+      <p class="name">Nicholas Engelbert</p>
+      <p class="contact">
+        <i class="fab fa-github"></i>
+        <a href="https://github.com/nicholas1301 target="_blank">nicholas1301</a>
+        <i class="fab fa-twitter"></i>
+        <a href="https://twitter.com/nicholas_eng" target="_blank">nicholas_eng</a>
+        <i class="fab fa-linkedin"></i>
+        <a href="https://www.linkedin.com/in/nicholas-engelbert-40b19b228/" target="_blank">Nicholas Engelbert</a>
+      </p>
+      <p class="about">Physics graduate and tech enthusiast making a career switch to the wonderful field of 
+        data science and machine learning. My resource recommendations are all related to these fields.</p>
+      <div class="resources">
+        <p>3 Useful Dev Resources</p>
+        <ul>
+          <li>
+            <a href="https://www.kaggle.com/" target="_blank" title="First Resource">
+              Kaggle
+            </a>
+          </li>
+          <li>
+            <a href="http://mlcourse.ai/book/index.html" target="_blank" title="Second Resource">
+              Free machine learning course
+            </a>
+          </li>
+          <li>
+            <a href="https://www.3blue1brown.com/" target="_blank" title="Third resource">
+              3Blue1Brown
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <!-- ________*END TEMPLATE* Contributor card END ________  --> 
```

Comment:
Change this line to 
`<!-- ________Nicholas' card END ________ -->`
It would make things easier for future contributors.